src/scripts/webpack/webpack.base.js

```diff
@@ -13,6 +13,8 @@ const JSLoader = t.enums.of(['babel', 'typescript'], 'JSLoader');
 
 export default ({ config, paths, NODE_ENV, jsLoader = JSLoader('babel') }) => {
 
+  process.env.config = JSON.stringify(config.bundle)
```

Comment:
I think you need to add this variable to the `DefinePlugin` instead (`process.env` of the build process is unrelated)? And also you can avoid to `JSON.parse` in the require after that, and the require altogether? It could be a "globalone" like `process.env.config` or even `appConfig` directly

Reply:
> And also you can avoid to JSON.parse in the require after that
the problem is typescript, but yeah, we could easily  cast